src/main/java/org/xstefank/check/TemplateChecker.java

```diff
@@ -31,22 +31,23 @@ public TemplateChecker() {
     }
 
     public void checkPR(JsonNode payload) {
-        StringJoiner joiner = new StringJoiner(", ");
+        //StringJoiner joiner = new StringJoiner(", ");
```

Comment:
Please do not commit commented code. In version control we can see what code was there before :).